submissions/yhorodechna/side-menu-dom/js/script.js

```diff
@@ -0,0 +1,107 @@
+let _navEl;
+const getNavElement = () => {
+    if (!_navEl) {
+        _navEl = document.querySelector('#nav');
```

Comment:
why not leave just this part? All that this function did is getting an element from DOM if it exists, right?

Reply:
Changed.